spec/unit/packagers/msi_spec.rb

```diff
@@ -316,6 +316,23 @@ module Omnibus
       end
     end
 
+    describe '#fast_msi' do
```

Comment:
The primary path that fastmsi impacts is functional - the setup and build steps, which don't currently have tests. If we had those yes we could run both modes.